.gitignore

```diff
@@ -1,6 +1,9 @@
 *.swp
 *.beam
+*.bk
 .DS_Store
+**/src
```

Comment:
Odd. Looking at the github files, I only see `src` in the Forth exercise. 

the `src/lib.rs` file in Forth sets up an Enum and stubs out some functions. That's something we talked about doing in a `hints.md` file in #117.

So we could move that forth file to a `hints.md` file and settle on a standard of "don't provide a stub src/lib.rs file". Or we could follow Forth's example and try to solve #117 by providing stub functions.

For student ease, I think the stubbed out `lib.rs` file is best. There are several exercises that require students to either comment-out tests or write their own stubs just so that they can get the first test to pass.

For example, Allergies. If you focus on the first test and implement `is_allergic_to`, then your tests will fail because you haven't yet implemented `allergies`. Even though that function isn't even introduced until the 4th test.

A stub file would solve that problem, right?